frontend/www/js/omegaup/components/common/Paginator.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div>
+    <div class="pager-bar">
+      <center>
+        <ul class="pagination">
+          <li v-for="page in pagerItems" v-bind:class="page.class">
+            <a
+              v-bind:href="page.url"
+              v-bind:class="{ disabled: page.class !== 'active' }"
+              >{{ page.label }}</a
+            >
+          </li>
+        </ul>
+      </center>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { omegaup, T } from '../../omegaup.js';
```

Comment:
nit:

```suggestion
import { omegaup, T } from '../../omegaup';
```

así haces el import en modo TypeScript (y es consistente con los demás archivos).

Reply:
Listos